common/test/acceptance/pages/studio/utils.py

```diff
@@ -144,7 +144,11 @@ def set_input_value(page, css, value):
     input_element.click()
     # Select all, then input the value
     input_element.send_keys(Keys.CONTROL + 'a')
-    input_element.send_keys(value)
+    # To allow an empty values we send DELETE key
+    if value == '':
+        input_element.send_keys(Keys.DELETE)
```

Comment:
> It removes all characters or 1? If the caret is at the beginning of the field it removes nothing. Am I correct?

It removes all characters because of `input_element.send_keys(Keys.CONTROL + 'a')`